packages/dashboard/components/ranger-dashboard.tsx

```
@@ -553,6 +555,47 @@ export default function RangerDashboard() {
+          s.lng > mapBounds.east
+        ) return false;
+      }
+      return true;
+    });
+  }, [mapSightings, mapSeverityFilter, mapTimeRange, mapBoundsActive, mapBounds]);
+
+  // re-fit map when filters change (skip initial mount)
+  useEffect(() => {
+    if (!mapFilterMountedRef.current) { mapFilterMountedRef.current = true; return; }
+    setMapFitKey((k) => k + 1);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)